03_how_ln_works.asciidoc

```diff
@@ -89,518 +93,526 @@ Also, in case you are not familiar with P2PKH addresses and the basic format and
+
+Now that we understand commitment transactions, let's look at some of the more subtle details. You may notice that this protocol leaves a way for either Alice or Bob to cheat.
+
+==== Cheating with prior state
+
+How many commitment transactions does Alice hold after she pays 3 mBTC to Bob? She holds two: the original one paying her 10 mBTC and the more recent one, paying her 7 mBTC and Bob 3 mBTC.
+
+In the channel protocol we have seen so far, nothing stops Alice from publishing a previous commitment transaction. A cheating Alice could publish the commitment transaction which grants her 10 mBTC.
+Since that commitment transaction  was signed by Bob he can't prevent Alice from transmitting it.
+
+Some mechanism is needed to prevent Alice from publishing an old commitment transaction. Let us now find out how this can be achieved and how it enables the Lightning Network to operate without requiring any trust between Alice and Bob.
+
+Because Bitcoin is censorship resistant, no one can prevent someone from publishing an old commitment transaction. To prevent this form of cheating, we construct commitment transactions so that if an old one is transmitted, the cheater can be punished. By making the penalty large enough, we create a strong incentive against cheating and this makes the system secure.
+
+The way the penalty works is by giving the cheated party an opportunity to claim the balance of the cheater. So if someone attempts to cheat by broadcasting an old commitment transaction, in which they are paid a higher balance than they are due, the other party can punish them by taking *both* their own balance and the balance of the cheater. The cheater loses everything.
+
+Let us go through the channel construction scenario again, adding a penalty mechanism to protect against cheating:
+
+* Alice creates a channel with Bob and put 10 mBTC into it.
+* Alice send 3 mBTC to Bob.
+* Alice tries to cheat Bob out of his earned 3 mBTC by publishing an old commitment transaction claiming the full 10 mBTC for herself.
+* Bob detects the fraud and punishes Alice by taking the full 10 mBTC for himself.
+* Bob ends up with 10 mBTC, gaining 7 mBTC for catching Alice cheat.
+* Alice ends up with 0 mBTC.
+* Trying to cheat Bob out of 3 mBTC, she loses the 7 mBTC she owned.
+
+With a strong penalty mechanism, Alice is not tempted to cheat by publishing an old commitment transaction as she risks losing her entire balance.
+
+[NOTE]
 ====
 In Mastering Bitcoin Andreas states it very elegantly:
 A key characteristic of bitcoin is that once a transaction is valid, it remains valid and does not expire. The only way to cancel a transaction is by double-spending its inputs with another transaction before it was mined. That's why we used timelocks [...] to ensure that more recent commitments could be spent before older commitments were valid.
```

Comment:
remove the second half of the quote. put the quote in quotations and then write something along the line. 

That is why we use timelocks on the to_local output to give the person that has the right to execute the penalty transaction a leanway. 

or take out the entire note